persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -88,10 +89,10 @@ public void execute() {
         }
 
         if (datastore == null) {
-            datastore = KEYWORD_MYSQL;
-        } else if (!datastore.equals(KEYWORD_MYSQL)) {
+            datastore = KEYWORD_IN_MEMORY_TABLE;
+        } else if (!datastore.equals(KEYWORD_MYSQL) && !datastore.equals(KEYWORD_IN_MEMORY_TABLE)) {
```

Comment:
This validation is growing, not look correct to handle each separately. Shall we use the set https://github.com/ballerina-platform/persist-tools/pull/190/files#diff-248599045e35b81e614c037fc726a8e7eaf1b66cc52d55e79e3712000e89e015R61

and check whether the value contains in the list?